help/search/_posts/2015-02-09-search-syntax.md

```diff
@@ -61,6 +61,21 @@ aff:(China OR "Hong Kong" Taiwan) | aff:((China OR "Hong Kong") AND Taiwan)
 aff:(China OR "Hong Kong" NOT Taiwan) |aff:(China OR ("Hong Kong" NOT Taiwan))
 aff:(China OR "Hong Kong" -Taiwan) | aff:((China OR "Hong Kong") NOT Taiwan)
 
+For a more heavy handed exploration of the search syntax, feel free to visit [search parser details](help/gory_details/_posts/2021-05-23-search-parser.md).
+
+
+### Wildcards, Proximity, and Regular expression search
+
```

Comment:
I've disabled that against unfielded search because it it mind-boggling the expansions that have to happen (when all synonyms and possible expansions are taken into account)

for virtual fields it works everywhere (I've tested it); so I prefer to leave it as it -- basically, you can use those operations in any fielded query